test/e2e/tools.mjs

```diff
@@ -30,6 +31,12 @@ export async function getStats(url) {
     return fs.readJSON(path);
 }
 
+export async function getApifyToken() {
+    const authPath = join(homedir(), '.apify', 'auth.json');
+    const { token } = await fs.readJSON(authPath);
```

Comment:
we might want to check if the file exists first (and throw a hint that login is required to run e2e tests), this would probably fail if not